index.js

```diff
@@ -0,0 +1,84 @@
+import updateClock from './moduls/luxon date.js';
+import Event from './moduls/eventlisenter.js';
+
+export default class BookCollection {
+  static initialize() {
+    const bookCollection = new BookCollection();
+    return bookCollection;
+  }
+
+  constructor() {
+    this.bookList = document.getElementById('bookList');
+    this.books = [];
+    this.loadBooks();
+    updateClock();
+    this.event = new Event(this);
+  }
+
+  saveBooks() {
+    localStorage.setItem('books', JSON.stringify(this.books));
+  }
+
+  displayBooks() {
+    this.bookList.innerHTML = '';
+
+    this.books.forEach((book, index) => {
+      const listItem = document.createElement('li');
+      const listItemH2 = document.createElement('h2');
+      listItem.appendChild(listItemH2);
+      // create an specific css class to alternate the style of the list
+
+      if (index % 2 !== 0) {
+        listItem.classList.add('makeMeGrey');
+      }
+      listItem.innerHTML = `${book.title} by ${book.author}`;
+
+      const removeButton = document.createElement('button');
+      removeButton.classList.add('removeButtonStyle');
+      removeButton.textContent = 'Remove';
+      removeButton.addEventListener('click', () => {
+        this.removeBook(index);
+      });
+
+      listItem.appendChild(removeButton);
+      this.bookList.insertBefore(listItem, this.bookList.firstChild);
+    });
+  }
```

Comment:
[OPTIONAL] Good job so far, :clap: , However, It will be very great if you could refactor this line of code and place it in a different file and use a functional approach then import it accordingly. :slightly_smiling_face: